### PR TITLE
Clockwork Cult rework

### PR DESCRIPTION
**Pull request by KeRSedChaplain**
## About The Pull Request
Makes Clockwork Cult less of a clusterfuck of plasma flooding and Gygax rushing (though I know the latter won't be stopped regardless)
This is finally in somewhat of a testmergable state, everything seems to be working on local.

## Why It's Good For The Game
Reebe was a garbage attempt at making clockwork cult unique from Blood Cult, but hopefully, this attempt will be able to capture what made both component-based clock cult AND Reebe clock cult fun.

## Changelog
:cl:
add: The belligerent scripture and a brass multitool, and a new marauder variant which act similar to holoparasites/guardian spirits.
del: Removed the abductor teleport consoles they get, removes abscond for the time being as I've not seen much use for it other than just spamming it and hoping you end up in the armory.
tweak: moved around scriptures to make the cult work better as being based around the station, makes the Ark scream more often and work as a summonable object, clockwork armor now has a flat 0 defense up to 10 instead of negatives against laser damage. Makes the Ark work better in a station based setting, as well as the Heralds beacon in case It works for the mode.
soundadd: added powerloaderstep.ogg for Neovgre
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
Good PR.

**Comment by KeRSedChaplain** [comment]
hrrng the Ark is getting stuck in a loop of activation byond why

File: code/modules/antagonists/clockcult/clock_mobs/clockwork_marauder.dm [hunk 1]
```diff
@@ -124,3 +124,435 @@
...
+		return FALSE
+	return ..()
+
+/mob/living/simple_animal/hostile/clockwork/marauder/guardian/bullet_act(obj/item/projectile/Proj)
+	if(blockOrCounter(null, Proj))
+		return
+	return ..()
+
+/mob/living/simple_animal/hostile/clockwork/marauder/guardian/hitby(atom/movable/AM, skipcatch, hitpush, blockedatom/movable/AM, datum/thrownthing/throwingdatum)
+	if(blockOrCounter(null, AM))
```
Comment (timothyteakettle) [inline code/modules/antagonists/clockcult/clock_mobs/clockwork_marauder.dm:365]: it should be blocked not blockedatom/movable/AM
you already defined atom/movable/AM

**Comment by KeRSedChaplain** [comment]
praise be to timothy for the assistance

**Comment by KeRSedChaplain** [comment]
realized halfway through that push it'd probably just error

**Comment by KeRSedChaplain** [comment]
ugh how long will the limbo of "everyone makes a PR that touches everything your touching" last

**Comment by KeRSedChaplain** [comment]
FUCK ME I HIT PUSH BY ACCIDENT

**Comment by KeRSedChaplain** [comment]
time to leave a comment on the PR Skyratbot is gonna make 🤡

**Pull request merged**